All volunteers email for start of competition cycle (#44)
srobo/volunteer-emails: SR2022/2021-09-22-start-of-comp-cycle.md

As you may already have seen, we have five main teams within the organisation. If any of these sounds interesting then please feel free to join one of the regular meetings in the [Google Calendar](https://calendar.google.com/calendar/embed?src=studentrobotics.org_oqdjasvpps8smo0d5nte417rak%40group.calendar.google.com&ctz=Europe%2FLondon)
and check out the relevant channels in our [Slack](SLACK LINK HERE).
We appreciate any help you can provide, whether that's taking on a couple of tasks or just joining the occasional meeting to say hello and get a feel for what's going on.

### Kit Team

Supporting the current kit which is provided to competitors and developing the next generation.
Led by: Karina Kwiatek, Dan Trickey

### Competition Team

Shaping this year's competition including the game, the venue, and managing the teams.
Led by: Andy Busse, Andy Barrett-Sprott, Jenny Fletcher

### Marketing Team

Promoting Student Robotics to Sponsors, Volunteers, and Competitors.
Led by: Antoine Petty, James Seden Smith

### Fundraising Team

Engaging with current and prospective sponsors of Student Robotics.
Key members: Jenny Fletcher, Holly Holder

### Infrastruture Team

> Supporting the tools used by us blueshirts.

Reviewer: Using the wording lifted from the in-progress ops manual change:

```suggestion
Maintain the technical infrastructure required to support the activities of Student Robotics.
```

Led by: Peter Law, Jake Howard

## Student Robotics 2022 Competition

Now, while we've still got a fair bit of work to do, here are our plans for the year ahead.

> * We're going to return to a more normal looking competition, with physical robots back this year.

Reviewer: ```suggestion
* We're planning on a return to a more normal looking competition, with physical robots back this year.
```

* We're expecting to run an online kickstart in late October / early November (i.e. after half term).
* This also means the challenge will be designed to fit within a classroom sized space.
* The simulator-based activities have worked rather well over the last couple of years - this will be included in a meaningful way into this year's competition.

### Mentoring

Our teams have had a couple of years of purely virtual competitions so may need a little more support getting to grips with our kit and robot-building again. 
This is an easy way to make a big difference for our competitors to boost their skills and confidence, even if you're fairly new to volunteering.
If you'd like to get involved, please let the competition team know!

## Website Overhaul

We're in the process of giving our website a fresh lick of paint (you may have already seen our [new sponsor page](https://studentrobotics.org/sponsor/) ).
If you have an interest or experience with design or video editing, the marketing team would love to hear from you. You don't need any particularly technical skills to get involved. :)

We look forward to seeing you soon

> -- SR Competition Committee

Reviewer: ```suggestion
-- SR Marketing Committee
```
